templates/default/quicksearch-indicators-panel.html

```diff
@@ -11,6 +11,13 @@
 		</div>
 	</div>
 	{/block}
+    <div style="margin-left: 10px;">
```

Comment:
Nie podoba mi się hardcodowanie styli graficznych.
Jeśli  już to przynajmniej na początku szablonu.

Reply:
Masz rację, przeniosłem na początek szablonu.